app/controllers/my_facilities_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class MyFacilitiesController < AdminController
+  include DistrictFiltering
+  include Pagination
+  include SizeFiltering
+
+  before_action :authorize_my_facilities, only: %i[index ranked_facilities blood_pressure_control]
+
+  def index
+    @users_requesting_approval = policy_scope([:manage, :user, User])
+                                 .requested_sync_approval
+                                 .order(updated_at: :desc)
+
+    @users_requesting_approval = paginate(@users_requesting_approval)
+
+    @facilities = policy_scope([:manage, :facility, Facility])
+    @inactive_facilities = MyFacilitiesQuery.inactive_facilities(@facilities)
+
+    @facility_counts_by_size = { total: @facilities.group(:facility_size).count,
+                                 inactive: @inactive_facilities.group(:facility_size).count }
+
+    @bp_counts = { last_week: @inactive_facilities.bp_counts_in_period(1.week.ago, Time.current),
```

Comment:
Are these BP counts for inactive facilities? If so, could we indicate that in this var name? eg.  `@inactive_facility_bp_counts`?